pkgs/tools/admin/azure-cli/extensions-manual.nix

```diff
@@ -9,8 +11,21 @@
     url = "https://github.com/Azure/azure-devops-cli-extension/releases/download/20240206.1/azure_devops-${version}-py2.py3-none-any.whl";
     sha256 = "658a2854d8c80f874f9382d421fa45abf6a38d00334737dda006f8dec64cf70a";
     description = "Tools for managing Azure DevOps";
+    propagatedBuildInputs = with python3Packages; [ distro ];
+  };
+
+  rdbms-connect = mkAzExtension rec {
+    pname = "rdbms-connect";
```

Comment:
As these extensions can't be manually updated, could you add yourself as maintainer?

Reply:
Sure, added myself to the list.